core/domain/feedback_services.py

```diff
@@ -187,9 +207,11 @@ def create_message(
     # the last_updated time changes to show that there is activity in the
     # thread.
     if thread.has_suggestion:
-        # TODO (nithesh): remove manual construction of suggestion ID after
-        # migrating feedback threads.
-        suggestion_id = 'exploration.' + thread_id
+        if feconf.ENABLE_GENERALIZED_FEEDBACK_THREADS:
+            suggestion_id = thread_id
+        else:
+            suggestion_id = 'exploration.' + thread_id
```

Comment:
Here and everywhere else: let us be consistent with the usage of `'exploration.'`. At some places, we are using the constant defined in feconf, while at other places we are just using `'exploration.'`. Let us use feconf.ENTITY_TYPE_EXPLORATION throughout.

Reply:
Replaced throughout the code